pkg/v6/resource/chartconfig/desired.go

```diff
@@ -68,7 +68,7 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 func (r *Resource) newIngressControllerChartConfig(ctx context.Context, clusterConfig cluster.Config) (*v1alpha1.ChartConfig, error) {
 	chartName := "kubernetes-nginx-ingress-controller-chart"
-	channelName := "0-1-stable"
+	channelName := "0-2-stable"
```

Comment:
This is just needed because I bumped the version to 0.2.0 as part of my migration changes.

Reply:
Out of curiosity, if this hardcoded or just temporary?
I thought chart operator pull the latest app published in the registry?

Reply:
chart-operator deploys patch versions automatically. The problem here is I bumped the minor version because of my migration changes. So the channel needs to be updated in the CRs.

Longer term for Service Catalog I'd like the charts to be in a YAML file like the releases YAML we use for operators.

Reply:
ok understood :thx: